### DISCUSSION TITLE
GA4 import

### DISCUSSION THREAD

**Discussion by boy51** [plausible/analytics#3240] [Anything Else]
Hi,

Sifting through the docs&discussions I couldn't find much info on this.

we are trying to import from google, but we have the issue that we have no view to choose.

![image](https://github.com/plausible/analytics/assets/54218424/09c4977d-4e28-470a-9704-ac124ddd9826)

There are some resources on this topic (i.e. https://www.youtube.com/watch?v=lizktxaQSJY).

Eventually I found out that we are already using GA4 and there is no way to go back now obviously.

I saw some [people mention earlier that](https://github.com/plausible/analytics/discussions/1794#discussioncomment-2875746) GA4 import is not supported.

So now UA is not supported anymore, is GA4 import a thing?

Sorry if I missed this discussion somewhere.

**Comment by metmarkosaric** [comment]
hi @boy51! basically when that view is empty, Google is not sending us any valid properties. our Google Analytics import works for UA properties only at the moment as that's the main product that people want to switch away from. we'd like to build a GA4 import in the future but cannot make any promises on when this will be done at this moment. we'll make it so that you can import your GA4 data into any existing Plausible dashboard when this is released so do keep your GA4 property for now. thanks!

**Comment by Alex-Hay** [comment]
Now that UA has been defunct for a good few months – it'd be great to get GA4 imports rolled out. Personally – we'd like to get more clients on Plausible, but at the moment, there will be a 2-3 months hole in their data. Fathom analytics have supported GA4 and UA imports for a while now.

**Comment by pau1phi11ips** [comment]
> we'd like to build a GA4 import in the future but cannot make any promises on when this will be done at this moment.

@metmarkosaric Any news on the GA4 importer?

**Comment by metmarkosaric** [comment]
**Update:** The GA4 import is ready! You can see how it works here: https://plausible.io/docs/google-analytics-import

It’s now possible to import both UA and GA4 properties into the same Plausible dashboard. We’ll automatically detect time ranges to prevent double counting of periods and to avoid any gaps in the data.

The same considerations apply to the GA4 import as to the UA import: Imported data is less functional than native data. It cannot be filtered, exported and some metrics may be missing due to limitations with the Google APIs. More details in the link above.

**Reply by pau1phi11ips** [reply]
@metmarkosaric awesome thanks for the update. Do I need to be on version 2.1 to get this in the Community Edition?

**Reply by ruslandoga** [reply]
v2.1.0-rc.1 includes GA4 imports: https://github.com/plausible/analytics/discussions/4084

**Reply by pau1phi11ips** [reply]
@ruslandoga thanks, I think we'll wait for the 2.1 stable as we've got over 1 Billion rows of UA data that I really don't want to risk messing up but that's great to hear it's coming to the CE edition.
